_config.yml

```diff
@@ -17,7 +17,7 @@ title: ChickTech Favorite Foods
 email: bihan.zh@gmail.com
 description: >- # this means to ignore newlines until "baseurl:"
   What are some of ChickTechRDU's favorite foods?
-baseurl: "" # the subpath of your site, e.g. /blog
+baseurl: ""
```

Comment:
Try to put the comment back on this line so it looks exactly the same as it did before. Thanks!